src/actions/index.js

```diff
@@ -0,0 +1,28 @@
+export const loginHandler = (token) => {
+  localStorage.setItem("token", token);
+  return {
+    type: "LOGIN",
+    payload: {
+      token: localStorage.getItem("token"),
```

Comment:
```suggestion
      token
```
곧바로 token을 전달해줘도 될 것 같습니다.

Reply:
수정했습니다!